src/main/java/org/mule/extension/mulechain/internal/helpers/ResponseHelper.java

```diff
@@ -22,19 +22,21 @@ private ResponseHelper() {}
   public static Result<InputStream, LLMResponseAttributes> createLLMResponse(String response,
                                                                              dev.langchain4j.service.Result<?> result,
                                                                              Map<String, String> responseAttributes) {
-    TokenUsage tokenUsage = result != null ? new TokenUsage(result.tokenUsage().inputTokenCount(),
-                                                            result.tokenUsage().outputTokenCount(),
-                                                            result.tokenUsage().totalTokenCount())
+
+    TokenUsage tokenUsage = result.tokenUsage() != null ? new TokenUsage(result.tokenUsage().inputTokenCount(),
```

Comment:
We can have a add condition for null check

result != null && result.tokenUsage() != null ... (rest)